prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -144,10 +139,52 @@ class FHIRConverter(
                     ).serialize()
                 )
             }
-        } catch (e: IllegalArgumentException) {
+        }
+    }
+
+    /**
+     * Converts an HL7 message into FHIR bundles
+     *
+     * [message] incoming HL7 message to be converted into FHIR
```

Comment:
Change this kdoc, so the args are inline with the rest of the paragraph.  This is the way the program wants to do it, and yes, there are older examples in many places.